### PR TITLE
feat: generate EN translations from FR file

### PR DESCRIPTION
**Pull request by AntoineAugusti**
At the moment we leverage 2 CSV files for translations on the admin:
- a French one
- an English one

The English one can be derived from the French automatically but this is not done. At the moment we spend a lot of time copy/pasting strings from one file to another.

Let's make this a tiny bit better and remove the need for the `en.csv` file. This file is generated based on: a new file, `en_base.csv` with some English overrides and then leverages keys from `fr.csv`.

Trello card: https://trello.com/c/yE8y27Ul/428-improve-translation-process-on-the-admin

### CODE REVIEW

File: scripts/generate_en_translations.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import csv
+from pathlib import Path
+
+root_folder = Path(__file__).parent.parent.absolute()
+translation_folder = "app/translations/csv"
```
Comment (jimleroyer) [inline scripts/generate_en_translations.py:5]: The `pathlib` module seems quite handy. This above also works on Windows.

Comment (AntoineAugusti) [reply scripts/generate_en_translations.py:5]: Yeah, love it, the API is so nice to read.

File: scripts/generate_en_translations.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+fr_file = root_folder / translation_folder / "fr.csv"
+en_base_file = root_folder / translation_folder / "en_base.csv"
+en_file = root_folder / translation_folder / "en.csv"
+
+headers = ["source", "target"]
+
+with open(fr_file) as fr, open(en_base_file) as en_base, open(en_file, "w") as en:
+    reader = csv.DictReader(fr, fieldnames=headers)
```
Comment (jimleroyer) [inline scripts/generate_en_translations.py:14]: Can we rename to `fr_reader`? It would be more explicit for code readability to me.

Comment (AntoineAugusti) [reply scripts/generate_en_translations.py]: 💯 👍

**Review by jimleroyer — APPROVED** [review]
:shipit:

File: scripts/generate_en_translations.py [hunk 3]
```diff
@@ -14,7 +14,14 @@
     fr_reader = csv.DictReader(fr, fieldnames=headers)
     base_en_reader = csv.DictReader(en_base, fieldnames=headers)
 
-    # Write a few English overrides
+    # Write the few English overrides we have.
+    #
+    # Rows in `en_base.csv` are what an ideal translation system
+    # should be: they map keys to values, changing what's displayed
+    # in English when used in HTML templates.
```
Comment (jimleroyer) [inline scripts/generate_en_translations.py:21]: 👍

**Pull request merged**